src/Shared/Microsoft.DotNet.Scaffolding.Shared/Project/ProjectModifierHelper.cs

```diff
@@ -379,25 +380,41 @@ internal static bool FilterOptions(string[] options, CodeChangeOptions codeChang
 
         internal static async Task<Document> AddTextToDocument(Document fileDoc, IEnumerable<CodeSnippet> codeChanges)
         {
-            Document editedDoc = null;
-            if (fileDoc != null && codeChanges != null && codeChanges.Any())
+            if (fileDoc is null || codeChanges?.Any() != true)
```

Comment:
!codeChanges?.Any or codeChanges?.Any is false not working. Is it due to the bool?

Reply:
I'll revert that line